guides/common/modules/proc_installing-the-insights-advisor-engine-in-project-using-import-and-export.adoc

```diff
@@ -0,0 +1,30 @@
+[id="installing-the-insights-advisor-engine-in-project-using-import-and-export"]
```

Comment:
This looks like the same as https://github.com/theforeman/foreman-documentation/pull/3785/files#diff-a01ec44d19027af4555bb0c8f34ce68b4cb5715530bbb0401ab6bee2e068486fR1

Reply:
One is for Satellite and one is for Foreman.